python/private/py_wheel.bzl

```diff
@@ -304,13 +304,15 @@ def _input_file_to_arg(input_file):
     return "%s;%s" % (py_package_lib.path_inside_wheel(input_file), input_file.path)
 
 def _py_wheel_impl(ctx):
+    is_stamping = is_stamping_enabled(ctx.attr)
+
     abi = _replace_make_variables(ctx.attr.abi, ctx)
     python_tag = _replace_make_variables(ctx.attr.python_tag, ctx)
     version = _replace_make_variables(ctx.attr.version, ctx)
 
     filename_segments = [
         _escape_filename_distribution_name(ctx.attr.distribution),
-        normalize_pep440(version),
+        _escape_filename_segment(normalize_pep440(version)) if is_stamping else normalize_pep440(version),
```

Comment:
Something strange is going on, or I'm missing something.

The net code change is the same as before: `normalize(...) if ... else normalize(...)`

The test was updated, though? Which would mean it's working?? Is the test not covering something...?